db.json

```diff
@@ -27,10 +27,6 @@
     {
       "name": "Wurst",
       "id": 7
-    },
-    {
-      "name": "",
-      "id": 8
```

Comment:
I know that was a suggestion on the last pullrequest. Since the pullrequest was merged already, you could have ignored this. You could put the db.json on your gitignore (if the json-server works in heroku without it not being on github). That  prevents your pullrequest being cluttered by db.json changes 😄

Reply:
That's a good poin. I will think about it 👍